tools/spelling/spelling_dictionary.txt

```diff
@@ -401,7 +401,9 @@ XFF
 XML
 XN
 XNOR
+XRay
```

Comment:
This doesn't look right, https://docs.aws.amazon.com/xray/latest/devguide/ spells this X-Ray as the canonical name, no?